src/pretalx/sso_provider/providers.py

```diff
@@ -8,6 +8,7 @@
 from allauth.socialaccount.helpers import render_authentication_error
 from django.conf import settings
 from django.urls import reverse
+from urllib.parse import urlencode
```

Comment:
Move this import line to the 1st group. [PEP8 coding style](https://peps.python.org/pep-0008/#imports):

> Imports should be grouped in the following order:
>
>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.